modules/install/pages/install.adoc

```diff
@@ -1,71 +1,538 @@
+You can install JBoss EAP 8.0 by using one of the following installation methods:
 
-== Pre-requisites
-
-Talk about what pre-requisites are needed:
+* *JBoss EAP using the Zip file*
+* *JBoss EAP using archive installation method*
+* *JBoss EAP by using the RPM installation method*
+* *JBoss EAP using the JAR installer*
 
-* Red Hat supported OpenJDK 17 or higher (21 recommended?)
-* Recommended to create a separate *jboss* user account with no shell access (/sbin/nologin) 
-* Recommended to install under /opt. Can be a dedicated partition. Ensure there is enough space (suggest a good minimum number)
+Let us see each of these methods in detail.
 
 == Installing JBoss EAP using the Zip file
 
-Brief overview only. Details in lab section below....
+* In this method, the JBoss EAP 8 is distributed as a zip file that you can extract and execute on an operating system (OS) with  OpenJDK 17 installed.
+* The zip file contains all the libraries, configuration files, and scripts needed to start JBoss EAP, *without* further customization.
+* This method is preferred when:
+** You don't have GUI support for the EAP 8 installation.
+** You don't want to have any custom configuration.
+** You need text based console for the EAP 8 installation.
+* Due to its portable nature, you can use it on multiple platforms(like Windows, Linux, etc.) and it requires OpenJDK 17 installed on the host OS.
+
+== Installing JBoss EAP using archive installation method
+
+* In this method, the JBoss EAP 8 is distributed as an archive file that you can extract and execute on an operating system (OS) with  OpenJDK 17 installed.
+* The archive file contains all the libraries, configuration files, and scripts needed to start JBoss EAP, *without* further customization.
+* Just like zip file method, this method is preferred when:
+** You don't have GUI support for the EAP 8 installation.
+** You don't want to have any custom configuration.
+** You need text based console for the EAP 8 installation.
+* Due to its portable nature, you can use it on multiple platforms(like Windows, Linux, etc.) and it requires OpenJDK 17 installed on the host OS.
+
+== Installing JBoss EAP by using the RPM installation method
+
+* JBoss EAP 8 is available as an RPM package for users with a valid subscription to the JBoss Enterprise Application Platform sub-channel, in the JBoss Enterprise Application Platform group.
+* The yum command downloads all the RPMs to the host OS, and automatically installs all the files needed to start and manage JBoss EAP.
+* The RPM installation method is only valid for Red Hat Enterprise Linux (RHEL) operating system and not for Windows server.
+* This method is preferred when you want JBoss EAP service files to be installed for the RHEL operating system.
 
 == Installing JBoss EAP using the JAR installer
 
-Brief overview only. Details in lab section below....
+* This method can also be called the GUI installer method.
```

Comment:
Add a sentence that this provides an "interactive" graphical approach where you answer a series of questions

Reply:
added the wordings